README.md

```diff
@@ -250,8 +255,16 @@ Nighthawk process. Note that increasing this results in an effective
 load multiplier combined with the configured --rps and --connections
 values. Default: 1.
 
+-p <http1|http2|http3>,  --upstream-protocol <http1|http2|http3>
```

Comment:
Is there value added by "upstream" here? Would just "protocol" perhaps be clearer to (a broader set of) end users?

Reply:
Done in ff921cf6de5db41ae5f85203622bc4d7a8a11c68.